lib/game/AdaptiveButton.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { useEffect, useState } from "react";
+import { TouchableOpacity, Text, View } from "react-native-web";
+import PropTypes from "prop-types";
+
+const AdaptiveButton = ({ action, noJsFallBack, text, style }) => {
```

Comment:
Not important but it would een more reusable if you used `children` instead of a `text` prop. Let me know if you want to change it or if I merge

Reply:
Hmmm really nice idea! I wasn't thinking of it... I'll do it right away, it's quick to implement